pkgs/tools/misc/flexoptix-app/default.nix

```diff
@@ -1,12 +1,12 @@
 { lib, appimageTools, fetchurl, nodePackages }: let
   pname = "flexoptix-app";
-  version = "5.11.0";
+  version = "5.12.1";
   name = "${pname}-${version}";
```

Comment:
That looks only slightly redundant.

Reply:
Not sure what you mean. I let both name and pname because I need both later

Reply:
Hrm. The `name` attribute is already exactly that concat, but it is not available within the derivation?

Reply:
Removed it as all appimage things now also take `pname`